install_config/web_console_customization.adoc

```diff
@@ -230,6 +230,33 @@ width and height. The ideal height is *20px*.
 Add the stylesheet as described in
 xref:loading-custom-scripts-and-stylesheets[Loading Extension Scripts and Stylesheets].
 
+[[changing-the-membership-whitelist]]
+== Customizing the Membership Whitelist
+
+The default whitelist in the membership page shows a subset of cluster roles, such as 
+`admin`, `basic-user`, `edit`, and so on. It also shows custom roles defined within a project.
+
+For example, to add your own set of custom cluster roles to the whitelist:
+
+[source, javascript]
+----
+window.OPENSHIFT_CONSTANTS.MEMBERSHIP_WHITELIST: [
```

Comment:
Hmm, we tried like yapei's https://github.com/openshift/openshift-docs/pull/9251#discussion_r193297113, `:` does not work, only `=` works, please help fix it. With this fixed, the doc will be fine